SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -1236,7 +1240,9 @@ void ActorSystem::ApplyComponentUpdate(const Worker_ComponentId ComponentId, Sch
 
 	ComponentReader Reader(NetDriver, RepStateHelper.GetRefMap(), NetDriver->Connection->GetEventTracer());
 	bool bOutReferencesChanged = false;
-	FObjectRepNotifies& ObjectRepNotifiesOut = RepNotifiesToSend.Emplace_GetRef(TWeakObjectPtr<UObject>(&TargetObject));
+	FObjectRepNotifies& ObjectRepNotifiesOut = TargetObject.IsA<AActor>()
+												   ? ActorRepNotifiesToSend.FindOrAdd(FWeakObjectPtr(&TargetObject))
+												   : SubobjectRepNotifiesToSend.FindOrAdd(FWeakObjectPtr(&TargetObject));
```

Comment:
could be refactored into a super-small helper, I guess

Reply:
done